runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/environment/DockerEnvironmentFactory.java

```diff
@@ -120,10 +125,10 @@ public RemoteEnvironment createEnvironment(Environment environment) throws Excep
     String provisionEndpoint = provisioningServiceServer.getApiServiceDescriptor().getUrl();
     String controlEndpoint = controlServiceServer.getApiServiceDescriptor().getUrl();
 
-    List<String> volArg =
-        ImmutableList.of(
-            // NOTE: Host networking does not work on Mac, but the command line flag is accepted.
-            "--network=host");
+    List<String> volArg = new ArrayList<>();
+    volArg.addAll(gcsCredentialArgs());
```

Comment:
I don't thinnk we _always_ want to mount this. Can you make doing so a job server command line arg and plumb it through?